src/app-layout/drawer.tsx

```diff
@@ -100,15 +85,10 @@ export function Drawer({
     <div
       className={clsx(styles.drawer, {
         [styles['drawer-closed']]: !isOpen,
-        [styles['drawer-hidden']]: isHidden,
         [styles['drawer-mobile']]: isMobile,
-        [styles['has-divider-with-splitpanel']]: hasDividerWithSplitPanel,
```

Comment:
`styles['drawer-hidden']` and `styles['has-divider-with-splitpanel']` do not exist. Removing